roadmap.md

```diff
@@ -42,9 +44,125 @@ What this vision looks like differs across different [users](user-profiles.md):
+
+#### Goal: Enable and Encourage Adoption
+
+Increase Tekton adoption and usage in alignment with our mission to provide “a
+powerful and flexible open-source framework for creating CI/CD systems, allowing
+developers to build, test, and deploy across cloud providers and on-premise
+systems.”
+
+* Identify and define critical user journeys.
+* Diversify available integrations with both OSS tools and vendor tools to
+  improve critical user journeys.
+* Investigate collection of Tekton usage metrics -- downloads, installations,
+  usage patterns -- based on website stats and/or CDF community collaboration
+  * Create new venues for user engagement (in addition to existing Slack)
+* Consider metric collection via CDF-owned Tekton user surveys 
+* Running Tekton at scale
+  * Performance benchmarking
+  * Documented production guidance
+  * Regular load testing
+* Documentation upgrades
+  * Onboarding documentation to ease the path for new adopters
+  * Common use cases and user journeys
+  * Documentation based on user profiles (platform engineer, application
+	engineer, cluster administrator, task developer)
+  * Running Tekton securely
+  * Running Tekton at scale
+  * Optimizing resource usage / cost optimization
+
+#### Goal: Develop new features and advance maturity of Tekton
+
+* Pipelines LTS promotions
+  * CSI volumes
+  * propagated parameters
+  * propagated workspaces
+  * auto-mapped workspaces
+  * task-level resource requirements
+  * Array and object parameters and results
+* Workflows
+  * Simplify SCM connection with pipeline execution
+  * Workflows Availability -- possibly via adoption of Pipelines as Code
+* Pipelines development
+  * Tekton Conformance definition
+  * Pipeline-level sidecars
+  * Pipelines in Pipelines to Alpha/Beta
+  * Integrated support for Artifacts as results
+  * Larger results and attestations in support of pipeline-level provenance
+    * Standardized, simplified, scalable data sharing across tasks
+  * (Progress on) Custom Tasks to GA
+  * [Default results](https://github.com/tektoncd/community/blob/main/teps/0048-task-results-without-results.md)
+  * Improved observability / pipeline tracing / distributed tracing
+  * User-facing observability tools
+* Catalog
+  * Inventory of Custom Tasks in Catalog to provide extensions (like manual approval task, remote execution task)
+  * ArtifactHub migration for the Tekton Hub
+  * Verified catalog tasks migrated to git based versioning
+  * Standardize use of Best Practices for Catalog Tasks
+  * Develop and document task creation and scripting
+  * Standardize task testing practices
+  * Ease contribution path for new Catalog contributors
+    * Uplevel documentation and how-to guides
+    * Provide a template for best practice adoption
+* Results
+  * Persistent Logs support
+* Dashboard
+  * Better integration with results
+  * Better integration with security features (signed resources, trusted tasks)
+* Operator
+* CLI
```

Comment:
Should we keep this bullet points even if empty?

Reply:
Reasonable point; I added notes around this.